docs/docsite/rst/guide_alicloud.rst

```diff
@@ -0,0 +1,128 @@
+Alibaba Cloud Compute Services Guide
+====================================
+
+.. _alicloud_intro:
+
+Introduction
+````````````
+
+Ansible contains several modules for controlling and managing Alibaba Cloud Compute Services (Alicloud).  The purpose of this
+section is to explain how to put Ansible modules together to use Ansible in Alicloud context.
+
+Requirements for the Alicloud modules are minimal.
+
+All of the modules require recent versions of footmark.  It can be installed from your OS distribution or python's "sudo pip install footmark" in your control machine.
```

Comment:
Do you what version is required?

Will the version in the different OS package managers be new enough?